### ISSUE TITLE
prepend error message as first element in specified row

### ISSUE THREAD

**Issue by streifi** [formvalidation/support#343]
I write a plugin to support forms build by the squarespace form builder.

Therefore i need to specify the row, but the error message should be the first child of the row, so that it is displayed above the input field.

So i need to prepend an element to the row instead of append it to the specified row.

What is the best way to achieve this.

Would be great if i can specify it in the plugin i wrote to support squarespace form builder.

**Comment by nghuuphuoc** [comment]
> So i need to prepend an element to the row instead of append it to the specified row.

The [Showing messages in custom area](http://formvalidation.io/examples/showing-message-custom-area/) might be useful to you.

**Comment by streifi** [comment]
Yes i know. But then i had to add above every field an additional element that i can return as container. I would be compatible with the existing squarespace form builder.

I would be great if you can add in the next release a new option for "appendTo" or prependTo"
and use this option in 
function _initField ...
.html(this._getMessage(field, validatorName))
                            .appendTo($message);
                        //add new option if it should be appentTo or prependTo
                            .prependTo($message);
